sklearn/decomposition/tests/test_fastica.py

```diff
@@ -388,3 +388,37 @@ def test_fastica_output_shape(whiten, return_X_mean, return_n_iter):
     assert len(out) == expected_len
     if not whiten:
         assert out[0] is None
+
+
+@pytest.mark.parametrize("add_noise", [True, False])
+@pytest.mark.parametrize("seed", range(2))
+def test_fastica_simple_different_solvers(add_noise, seed):
```

Comment:
We recently introduced a "global random seed":

```suggestion
def test_fastica_simple_different_solvers(add_noise, global_random_seed):
```

REF: https://scikit-learn.org/dev/computing/parallelism.html#sklearn-tests-global-random-seed